comp110-worksheet-6/OxoBoard.cs

```diff
@@ -10,49 +10,139 @@ public enum Mark { None, O, X };
 
 	public class OxoBoard
 	{
-		// Constructor. Perform any necessary data initialisation here.
-		// Uncomment the optional parameters if attempting the stretch goal -- keep the default values to avoid breaking unit tests.
-		public OxoBoard(/* int width = 3, int height = 3, int inARow = 3 */)
+        // Constructor. Perform any necessary data initialisation here.
+        // Uncomment the optional parameters if attempting the stretch goal -- keep the default values to avoid breaking unit tests.
+        Mark[,] board;
```

Comment:
Comment documenting the constructor has gotten separated from the constructor itself -- should be moved